rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly make sure to add an `:person`, and `:book` to the attr_accessor so it can be writeable also as readable